### PR TITLE
update the yaml of deployment and replicas

### PR DESCRIPTION
**Pull request by jianglingxia**
maybe -L -o to -Lo consistent  with the following context and update the yaml to k8s v1.9

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7107)

### CODE REVIEW

**Comment by zhangxiaoyu-zidif** [comment]
/lgtm
@heckj  for approve

**Comment by steveperry-53** [comment]
This statement is not accurate:
apps/v1 # for versions before 1.9.0 use apps/v1beta2

The statement says that we can use apps/v1beta2 for any version before 1.9. That says we can use apps/v1beta2 for 1.8, 1.7, 1.6, 1.5 etc. But v1/beta2 does not exist for all those versions.

**Comment by tengqm** [comment]
Agree to @steveperry-53 The comment should be as accurate as possible.

File: docs/tasks/federation/policy-engine-deployment.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```
Comment (tengqm) [inline docs/tasks/federation/policy-engine-deployment.yaml]: please remove the comment

File: docs/tasks/federation/replicaset-example-policy.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1 # for versions before 1.9.0 use apps/v1beta2
```
Comment (tengqm) [inline docs/tasks/federation/replicaset-example-policy.yaml]: pls remove the comment or make it more accurate.

**Comment by heckj** [comment]
/hold
@jianglingxia could you please make the updates that Steve and Qiming have recommended? The intent is good, but specifics incorrect.

**Comment by jianglingxia** [comment]
ok ,thanks all,I will correct it soon!

**Comment by jianglingxia** [comment]
In order to be consistent，so I delete the comment and update it,thanks! @heckj @tengqm @steveperry-53

**Pull request merged**